src/wrappers/foresight/lib/jest/ModuleLoader.ts

```diff
@@ -9,6 +9,12 @@ import Trace from '../../../../plugins/Trace';
 const shimmer = require('shimmer');
 const has = require('lodash.has');
 
+export let TransformWrapped = false;
```

Comment:
it might be better to export `isTransformWrapped` instead of exporting global `TransformWrapped` variable here